packages/shared/src/components/profile/AccountDangerZone.tsx

```diff
@@ -46,8 +51,10 @@ function AccountDangerZone({
       {children}
       <Button
         onClick={onDelete}
-        buttonSize={ButtonSize.Small}
-        className="self-start mt-6 btn-primary-ketchup text-theme-label-primary"
+        size={ButtonSize.Small}
+        variant={ButtonVariant.Primary}
+        color={ButtonColor.Ketchup}
+        className="self-start mt-6 text-theme-label-primary"
```

Comment:
Maybe I missed something like this discussed in buttonV2, or something like this was out of scope, so just checking. 
For `text-theme-label-primary` wouldn't we think a button with a certain color/variant combination has a set label theme color?
Could be something that we can improve later on, but I believe it would be great if a default color for a certain combination is already set and consumers don't have to define that all the time.

No need to take action on this on this PR just raising for discussion and further work improvements we can do on the button.

Reply:
Yes, that's already the case. In this case, it was overriden to white font, so I kept the same visuals as we had before. But agree, ideally we do not add such a classname and keep the default label color.

Reply:
Ah that's actually a good catch.
In the beginning the red color was always with white text we overruled that for accesibility concerns.
So technically you shouldn't have custom colors on buttons.

But maybe double check with design and give them a pre-after screenshot?